### ISSUE TITLE
Issue with PlotSquared w/ AcidIsland&ASkyBlock

### ISSUE THREAD

**Issue by TNTUP** [IntellectualSites/PlotSquared#1052]
**Debug paste link**:
http://hastebin.com/xererucaya

**Description of the problem:**
I've found an incompatibility with [USER=16926]@Tastybento[/USER]'s plugins such as ASkyBlock and AcidIsland. Without P2, his plugins works fine. With P2, its triggering something weird which causes new player join with a new inventory/location. Removing P2 fixes it.

BUT! This is backwards, redoing the same steps, now removing ASkyBlock and AcidIsland. Without them, players inventory and locations are fine. Loading them with P2 causes this problem.

No errors at this time while running them both. Can you look at this?

There's something which doesn't work well with P2 and TastyBento's plugins... I had them working fine in 1.8.8....

**How to replicate**:

ASkyBlock and AcidIsland are not active on the production serveur, but can easily replicate it.

Create a server with AcidIsland and ASkyBlock. Take note of your inventory and location. Stop the server. Get PlotSquared and start the server. Notice it will say "Welcome to our new player" message, and you'll notice you have a different inventory and location. Removing PlotSquared fixes it. BUT! I can do backwards by removing @TastyBento's plugins, notice your inventory is back. There's an incompatibility with those. They're fine since 1.8.8, never happened.

I've already posted a bug report to TastyBento, seems there's an incompatibilty somewhere. 

See this issue on his bug report: https://github.com/tastybento/askyblock/issues/206

**Comment by TNTUP** [comment]
Hey guys, I haven't got an update on this ticket, but I've did a video to show the issue (I did on a fresh install aswell!) Here's the link> https://www.youtube.com/watch?v=WUNY3yuGOlw (The video can look blurry, its processing at this time of writing)

Steps to reproduce:

1.Fresh install (Get Spigot or PaperSpigot)
2.Download Multiverse-Core, Vault, PlotSquared, AcidIsland and ASkyblock
3. Start the server without plugins then stop it.
4. Add AcidIsland and ASkyblock first and start the server. Do thing s and get stuff in your inventory. Disconnect and stop the server.
5. Add PlotSquared. Start the server and see you have an empty inventory w/ new location (the spawn)
6. Removing either PlotSquared or AcidIsland/ASkyblock fixes it.
7. But! PlotSquared does something weird, you see "BukkitPlotGenerator doesn't support ...null" Can it be the cause of it?
